test/integration/http2_integration_test.cc

```diff
@@ -1661,4 +1661,49 @@ TEST_P(Http2IntegrationTest, OnLocalReply) {
   }
 }
 
+static std::string remove_response_headers_filter = R"EOF(
+name: remove-response-headers-filter
+typed_config:
+  "@type": type.googleapis.com/google.protobuf.Empty
+)EOF";
+
+TEST_P(Http2IntegrationTest, HeadersOnlyRequestWithRemoveResponseHeadersFilter) {
```

Comment:
can these two tests (and their H/1 counterparts) just be moved to protocol_integration_test?

Reply:
Thanks for pointing out, and moved tests to protocol_integration_test matrix :)